### DISCUSSION TITLE
filter custom attributes on More-info dropdown

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#10167] [Dashboard]
please consider adding the option to filter custom attributes from the more-info dropdown. 

We can now easily create custom attributes (eg to facilitate templates using `selectattr()`) which don't need to be in the frontend dropdown at all. Those attributes are really only needed in the backend.

Core attributes are now filtered here: https://github.com/home-assistant/frontend/blob/7bc27082595f943d15ca35ae948c766184a93eed/src/util/hass-attributes-util.ts#L87 and it would be really nice to be able to set some custom attributes to be added to that list in a customize_glob:

```
hide_attributes:
  - integration # see https://community.home-assistant.io/t/is-there-a-way-to-determine-what-integration-an-entity-belongs-to/247069/5
  - zone
  - has_battery
  - package
  - etc
```
or notice why Petro does: https://community.home-assistant.io/t/add-bring-back-friendly-name-to-template-sensors/305717/14

for #hacktoberfest ;-)

**Comment by spacegaier** [comment]
With "dropdown" you mean the collapsible/expandable section at the bottom of the more-info dialog?

Not sure if that is something for vanilla frontend or not better suited for a custom UI element...

**Reply by Mariusthvdb** [reply]
Yes thats what I meant. Obviously I’ve been trying that in custom-ui custom card, without success … hence my request to the dev team.

seems appropriate with the core support for [attributes_templates](https://www.home-assistant.io/integrations/template/#attributes) et all.

**Comment by Mariusthvdb** [comment]
is there any chance we could get an additional attribute in that list to hide from displaying in the attributes dropdown panel?

it would make life for customizing much easier, and the frontend way prettier ;-)
